config/config-network.yaml

```diff
@@ -98,7 +98,8 @@ data:
     # entirely from the template. When choosing a new value be thoughtful
     # of the potential for conflicts - for example, when users choose to use
     # characters such as `-` in their service, or namespace, names.
-    #
+    # {{.Annotations}} can be used for any customization in the go template if needed. 
+    # Make sure you don't have dns-name conflict if you use it.
```

Comment:
Change to: we strongly recommend keeping namespace part of the template to avoid domain name clashes.